packages/playwright/src/isomorphic/testServerInterface.ts

```diff
@@ -81,13 +81,15 @@ export interface TestServerInterface {
     locations?: string[];
     grep?: string;
     grepInvert?: string;
+    onlyChanged?: string;
   }): Promise<{
     report: ReportEntry[],
     status: reporterTypes.FullResult['status']
   }>;
 
   runTests(params: {
     locations?: string[];
+    files?: string[]; // todo: find a better name. this is like locations, without pattern matching and without line numbers.
```

Comment:
I think we should not need this if we align with UI mode.

Reply:
If we align with UI mode's way of passing an exhaustive list of test IDs, then we wouldn't need it. In order to do so, it needs to walk the entire test tree though. That's complex. It's fine in the UI, because it allows watching only select tests. We don't need that functionality in CLI watch mode though.

We still need something to filter down the test run to dirty files, and this `files` property is an easy-to-understand way of doing that.